doc/source/whatsnew/v1.0.0.rst

```diff
@@ -221,8 +221,10 @@ Other enhancements
 - DataFrame constructor preserve `ExtensionArray` dtype with `ExtensionArray` (:issue:`11363`)
 - :meth:`DataFrame.sort_values` and :meth:`Series.sort_values` have gained ``ignore_index`` keyword to be able to reset index after sorting (:issue:`30114`)
 - :meth:`DataFrame.to_markdown` and :meth:`Series.to_markdown` added (:issue:`11052`)
-
 - :meth:`DataFrame.drop_duplicates` has gained ``ignore_index`` keyword to reset index (:issue:`30114`)
+- :meth:`DataFrame.sort_values`, :meth:`DataFrame.sort_index`, :meth:`Series.sort_index`, and :meth:`Series.sort_index`
```

Comment:
also i think its worthwhile to add a sub-section to showcase this (e.g. put above other enhancements)

Reply:
show a simple example, then point to the docs pages

Reply:
I added an example and then said to "see examples and documentation in :meth:`DataFrame.sort_values`".